src/include/wiredtiger.in

```diff
@@ -661,6 +661,18 @@ struct __wt_cursor {
 	 */
 	int __F(reconfigure)(WT_CURSOR *cursor, const char *config);
 
+	/*!
+	 * Get the largest key of the cursor regardless of visibility. Call WT_CURSOR::get_key to
+	 * retrieve the largest key. However, it throws if WT_CURSOR:get_value is called. The cursor
+	 * is not positioned after that. Only supported by data cursors.
+	 *
+	 * @snippet ex_all.c Reset the cursor
+	 *
+	 * @param cursor the cursor handle
+	 * @errors
+	 */
+	int __F(largest_key)(WT_CURSOR *cursor);
```

Comment:
I wouldn't mention `get_key` or `get_value`, that's standard stuff, right?

"Throws" doesn't make sense to me, there aren't any exceptions here?

The phrase "after that" isn't really defined? After the call? After the throw?

The snippet line is wrong, this isn't "Reset the cursor"

The error line should probably mention the timestamp error (and maybe that's where we can say `get_value` returns an error.